src/votypes.jl

```diff
@@ -99,6 +103,11 @@ function _parse_binary(::Type{Bool}, data)
     c in ('F', 'f', '0') && return false
     return missing
 end
+function _parse_binary(::Type{Vector{T}}, data) where {T}
+    nbyte = sizeof(T)
```

Comment:
Hmm... I don't know if it would work.  Such arrays would only be possible for `datatype='boolean'` as above, and I don't think that I have, or can easily construct, test data with mixed missing/boolean arrays.